assets/styles/about_us.scss

```diff
@@ -0,0 +1,307 @@
+    animation-name: pulse;
+    -webkit-animation-duration: 1s;
+    animation-duration: 1s;
+    -webkit-animation-fill-mode: both;
+    animation-fill-mode: both;
+}
+
+.linkedinProfil {
+    grid-row: 3/4;
+    grid-column: 3/4;
+    display: flex;
+    justify-content: center;
+    text-decoration: inherit;
+    color: var(--our-black);
+    border-radius: 0.625rem;
+    background: var(--main-color);
+    box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.25);
+    max-width: 255px;
+    min-width: 190px;
+    width: 90%;
+    max-height: 53px;
+    flex-shrink: 0;
+    align-items: center;
+    margin-right: 1rem;
+}
+
+.linkedinProfil:hover {
+    -webkit-animation-name: pulse;
+    animation-name: pulse;
+    -webkit-animation-duration: 1s;
+    animation-duration: 1s;
+    -webkit-animation-fill-mode: both;
+    animation-fill-mode: both;
+}
+
+.githubProfil {
+    grid-row: 3/4;
+    grid-column: 4/5;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: inherit;
+    color: var(--our-black);
+    max-width: 255px;
+    min-width: 190px;
+    width: 90%;
+    max-height: 53px;
+    border-radius: 0.625rem;
+    border: 1px solid var(--main-color);
+    background: rgba(255, 164, 88, 0.07);
+    box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.25);
+}
+
+.githubProfil:hover {
+    -webkit-animation-name: pulse;
+    animation-name: pulse;
+    -webkit-animation-duration: 1s;
+    animation-duration: 1s;
+    -webkit-animation-fill-mode: both;
+    animation-fill-mode: both;
+}
+
+article a img {
+    margin-right: 1rem;
+}
+
+/* Mobile */
+
+@media screen and (max-width:1100px) {
+    main {
+        margin: 0 2rem;
+    }
+
+    article .firstquote {
+        display: none;
+    }
+
+    article aside p span {
+        display: none;
+    }
+
+    article {
+        grid-template-columns: 1fr;
+        grid-template-rows: repeat(6, auto);
+    }
+
+    .profil {
+        display: flex;
+        justify-content: center;
+    }
+
+    article .profil img {
+        width: 13.875rem;
+        height: 13.875rem;
+        grid-column: 1;
+        grid-row: 1;
+    }
+
+    article h2 {
+        grid-row: 2;
+        grid-column: 1;
+        margin-bottom: 0.9rem;
+    }
+
+    .website {
+        grid-row: 3;
+        grid-column: 1;
+        margin-bottom: 4rem;
+    }
+
+    article aside {
+        grid-row: 4;
+        grid-column: 1;
+        margin-bottom: 2rem;
+    }
+
+    .linkedinProfil {
+        grid-row: 5;
+        grid-column: 1;
+        justify-self: center;
+        min-height: 50px;
+        margin-bottom: 1.3rem;
+        margin-right: 0;
+    }
+
+    .githubProfil {
+        grid-row: 6;
+        grid-column: 1;
+        justify-self: center;
+        min-height: 50px;
+    }
+
+}
+
+@media screen and (max-width:700px) {
+    .longSeparation {
+        display: none;
+    }
+
+    .smallSeparation {
+        display: flex;
+    }
+}
+
+@-webkit-keyframes pulse {
```

Comment:
if this pulse keyframe is used anywhere else than in this file, shift it into global declarations